cvmfs/cvmfs_server

```diff
@@ -4046,8 +4052,20 @@ snapshot() {
       -i ${spool_dir}/tmp/last_snapshot \
       -o .cvmfs_last_snapshot"
 
-    # remove magic file after successful replication
-    $user_shell "$delete_command"
+    # run the automatic garbage collection (if configured)
+    if is_garbage_collectable $name && [ ! -z "$CVMFS_GC_TIMESTAMP_THRESHOLD" ]
```

Comment:
Update: Added a [Pull Request](https://github.com/cvmfs/cvmfs/pull/694) that both renames `CVMFS_GC_TIMESTAMP_THRESHOLD` and removes the necessity of setting `CVMFS_GARBAGE_COLLECTION=true` on stratum 1s.